eng/pipelines/corefx-base.yml

```diff
@@ -84,36 +84,27 @@ jobs:
           - group: DotNet-Internal-Tools-Feed
           - group: DotNet-Blob-Feed
 
-        - ${{ if eq(parameters.isOfficialBuild, 'false') }}:
-          - _msbuildCommonParameters: ''
-
         - ${{ if eq(job.submitToHelix, 'true') }}:
           - group: DotNet-HelixApi-Access
           - _archiveTestsParameter: /p:ArchiveTests=Tests
 
+        - _commonArguments: -restore -build -configuration $(_BuildConfig) -ci -buildtests -arch $(_architecture) -framework $(_framework) $(_archiveTestsParameter)
+        - _windowsOfficialBuildArguments: ''
+
         # Windows variables
         - ${{ if eq(parameters.targetOS, 'Windows_NT') }}:
           - _buildScript: build.cmd
-          - _commonArguments: -restore -build -configuration $(_BuildConfig) -ci -buildtests -arch $(_architecture)
           - _msbuildCommand: powershell -ExecutionPolicy ByPass -NoProfile eng\common\msbuild.ps1 -warnaserror:0 -ci
-
           - ${{ if eq(parameters.isOfficialBuild, 'true') }}:
             - _windowsOfficialBuildArguments: -sign
                                               /p:DotNetSignType=$(_SignType)
 
-          - ${{ if eq(parameters.isOfficialBuild, 'false') }}:
-            - _windowsOfficialBuildArguments: ''
-
         # Non-Windows variables
         - ${{ if ne(parameters.targetOS, 'Windows_NT') }}:
           - _buildScript: ${{ job.buildScriptPrefix }}./build.sh
-          - _args: --restore --build --configuration $(_BuildConfig) --ci --buildtests --arch $(_architecture)
-          - _commonArguments: $(_args)
-          - ${{ if eq(parameters.isOfficialBuild, 'true') }}:
-            - _commonArguments: $(_args) --stripSymbols
-
           - _msbuildCommand: ${{ job.buildScriptPrefix }}./eng/common/msbuild.sh --warnaserror false --ci
-          - _windowsOfficialBuildArguments: ''
+          - ${{ if eq(parameters.isOfficialBuild, 'true') }}:
+            - _commonArguments: $(_commonArguments) -stripSymbols
```

Comment:
Is this supported? I didn’t know appending to the same variable in yaml (azure devops) was supported.

Reply:
Seems like you are reviewing an old commit.

Reply:
That's what happens when you review from the phone 😄